### ISSUE TITLE
Mod not working on 1.20.1

### ISSUE THREAD

**Issue by keanine** [PepperCode1/Continuity#394]
I've tried creating a new instance of 1.20.1 with only continuity and fabric API installed, enabled the built in resource pack and a CTM resource pack, and the mod just isn't working properly. The bookshelves and sandstone etc work from the default pack, but every other CTM pack I've tried hasn't worked. I've also tried it with sodium and indium and the resource packs still don't work.

I've included the log of a fresh instance with Fabric API, Continuity 3.0.0 - beta.4 1.20.1 and the resource pack CTM Overhaul by xokz

[latest.log](https://github.com/PepperCode1/Continuity/files/14725965/latest.log)

**Comment by PepperCode1** [comment]
Elaborate what you mean by "the resource packs... don't work." Show screenshots of blocks not connecting as you expected and explain what it should look like. If there are example screenshots on the resource packs' project pages, you can compare against them. I have personally tested CTM Overhaul before and it has worked.

**Comment by keanine** [comment]
I've just tried the same setup on a different computer and have the same issue. I can provide screenshots, however it will just look like vanilla Minecraft, as the CTM overlays don't appear to be functional at all. I've taken a screenshot of built in textures working next to the CTM Overhaul textures not working. I've also tried continuity many times before with CTM Overhaul and never had an issue until now. Please let me know if there is anything else I can provide.
<img width="960" alt="Untitled" src="https://github.com/PepperCode1/Continuity/assets/11248290/ae751566-5a1f-48c5-bbf7-0a5a57eac0a2">
<img width="960" alt="Untitled2" src="https://github.com/PepperCode1/Continuity/assets/11248290/6fa4d518-f1e0-44c9-a759-4be1d7af7044">

**Comment by keanine** [comment]
Ok it looks like I'm probably wrong. It seems like they are working and I was misunderstanding what the resource packs do. I'll confirm I'm wrong and then I will close this issue if I am, otherwise I'll provide more info

**Comment by keanine** [comment]
Apologies for the hassle, the mod is working as intended. I was under the impression that all the resource packs I tried (I tried 6 different overhauls in total) had support for grass/sand/gravel textures etc. This was not the case, despite my understanding that they used to be a very common feature of connected textures. I found an add-on for a resource pack that does indeed have the working textures I was looking for.

Thank you for your quick response, I'm sorry to have wasted your time and good luck with continued develoment
